<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEntryWidgetHistory</class>
 <widget class="QWidget" name="EditEntryWidgetHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="historyView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
